R/ggMarginalGadget.R

```diff
@@ -76,7 +77,7 @@ ggMarginalGadgetHelper <- function(origPlot, addin) {
   }
 
   if (!ggplot2::is.ggplot(get(plotname)) &&
-      !ggplot2::is.ggplot(get(plotname, envir = .GlobalEnv))) {
+    !ggplot2::is.ggplot(get(plotname, envir = .GlobalEnv))) {
```

Comment:
I assume this was done by the auto formatter. Seems like I don't agree with the autoformat 100% of the time :) No need to fix it I don't actually care much, just tried to find something to comment on!

Reply:
This was indeed  a `styler` call, but note that I did do some manual tweaks of the formatting so it was all consistent.